.specify/scripts/bash/check-prerequisites.sh

```
@@ -0,0 +1,166 @@
+            exit 0
+            ;;
+        *)
+            echo "ERROR: Unknown option '$arg'. Use --help for usage information." >&2
+            exit 1
+            ;;
+    esac
+done
+
+# Source common functions
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$SCRIPT_DIR/common.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./common.sh was not specified as input (see shellcheck -x).

(SC1091)